style.css

```diff
@@ -6,180 +6,163 @@
-  -webkit-transform: rotate(90deg);
-  transform: rotate(90deg);
-  top: 0;
-  
-}
-
-label input:checked + .menu + ul { opacity: 1; }
-
-/* link style */
-label ul {
-  font-family: 'Lexend', sans-serif; 
-  color: black;
-  font-size: 18pt;
-  font-weight: bold;
-  letter-spacing: 0.04em;
-  z-index: 200;
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  -webkit-transform: translate(-50%, -50%);
-  transform: translate(-50%, -50%);
-  opacity: 0;
-  -webkit-transition: .25s 0s ease-in-out;
-  transition: .25s 0s ease-in-out;
-
-}
-
-label a {
-  margin-bottom: 1em;
-  display: block;
-  color: black;
-  text-decoration: none;
-  
-}
-
-/*style for burger menu ends*/
+	background-color: blue;
+	/* Create the parallax scrolling effect */
+ 	background-attachment: fixed;
+  	background-position: center;
+ 	background-repeat: no-repeat;
+  	background-size: cover;
+	overflow-x: hidden;
+	margin: 0;
+	padding: 0;
+}
+
+
+/*style for nav menu starts*/
+.navbar{
+	box-sizing: border-box;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: white;
+}
+
+.navbar-links ul{
+	margin: 0;
+	padding: 0;
+	display: flex;
+}
+
+.navbar-links li{
+	list-style: none;
+}
+
+.navbar-links li a{
+	text-decoration: none;
+	padding: 1rem;
+	display: block;
+}
+
+/* burger menu icon that displays on smaller screen sizes*/
+.toggle-button{
+	position: absolute;
+	top: .75rem;
+	right: 1rem;
+	display: none; /* not default so it only displays on smaller screen sizes*/
+	flex-direction: column;
+	justify-content: space-between;
+	width: 30px;
+	height: 21px;
+}
+
+.toggle-button .bar{
+	width: 100%;
+	height: 3px;
+	border-radius: 10px;
+	background-color: black;
+}
+
+/* media query that defines what width the navbar changes into a burger menu*/
+@media (max-width: 600px){
+	.toggle-button{
+		display: flex;
+	}
+	
+	.navbar-links{
+		display: none;
+		width: 100%;
+	}
+	
+	.navbar{
+		flex-direction: column;
+		align-items: flex-start;
+	}
+	
+	.navbar-links ul{
+		width: 100%;
+		flex-direction: column;
+	}
+	
+	.navbar-links li{
+		text-align: center;
+	}
+	
+	.navbar-links.active{
+		display: flex;
+	}
+}
+/*style for nav menu ends*/
 
 /* style for image banner - used in query pages */
 .banner{
+	margin-top: -20px;
+	padding: 0;
 	background-color: grey;
-	height: 400px;
+	height: 250px;
+	width: 700px;
 	position: relative;
 }
 
-
 /* style for main content */
 .content{
 	box-sizing: border-box;
-	box-shadow: 5px 5px black;
 	background-color: white;
-	border-radius: 25px;
+	border: 10px solid black;
 	padding: 25px 25px;
 	display: block;
-	margin: 50px auto;
-	width: 100%;
-	height: 600px;
+	margin: 0 auto;
+	width: 700px;
+	height: 500px;
+	overflow: auto;
 	position: relative;
 }
 
-/* style for main content on admin pages */
-.admin_content{
-	box-sizing: border-box;
-	box-shadow: 5px 5px black;
+.scroll-trial{
+	height: 700px;
+}
+
+/* style for admin menu on admin pages starts*/
+.admin-menu{
```

Comment:
Changed styling of admin menu from being in aside to horizontally on top of main content on all admin pages